tests/src/system/basic/WskBasicTests.scala

```diff
@@ -165,17 +175,27 @@ class WskBasicTests
     }
 
     it should "reject bad command" in {
-        wsk.cli(Seq("bogus"), expectedExitCode = MISUSE_EXIT).
+        if (usePythonCLI) {
+            wsk.cli(Seq("bogus"), expectedExitCode = MISUSE_EXIT).
             stderr should include("usage:")
+        } else {
+            val result = wsk.cli(Seq("bogus"), expectedExitCode = ERROR_EXIT)
+            result.stderr should include regex ("""(?i)Run 'wsk --help' for usage""")
+        }
     }
 
     it should "reject authenticated command when no auth key is given" in {
         // override wsk props file in case it exists
         val wskprops = File.createTempFile("wskprops", ".tmp")
         val env = Map("WSK_CONFIG_FILE" -> wskprops.getAbsolutePath())
-        val stderr = wsk.cli(Seq("list"), env = env, expectedExitCode = MISUSE_EXIT).stderr
-        stderr should include("usage:")
-        stderr should include("--auth is required")
+
+        if (usePythonCLI) {
+            val stderr = wsk.cli(Seq("list"), env = env, expectedExitCode = MISUSE_EXIT).stderr
+            stderr should include("usage:")
+            stderr should include("--auth is required")
+        } else {
+            val result = wsk.cli(Seq("list"), env = env, expectedExitCode = UNAUTHORIZED)
```

Comment:
this suggests the CLI is actually make the API call without an auth token. yes? if so, why make a call known to fail?